### PR TITLE
Random images

### PR DESCRIPTION
**Pull request by elendil7**
First attempt at implementation of 100% unique random images. Not yet working properly.

### CODE REVIEW

File: resources/script/random_bg/randomImage.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+const fs = require('fs');
```
Comment (Tolfx) [inline resources/script/random_bg/randomImage.js:1]: This won't work, since it doesn't run in node.js as backend.

**Review by Tolfx — CHANGES_REQUESTED** [review]

File: resources/script/random_bg/randomImage.js (hunk 1 above)
Comment (elendil7) [reply resources/script/random_bg/randomImage.js:1]: I used browserify to emulate the nodejs for frontend, code is in bundle.js

Comment (Tolfx) [reply resources/script/random_bg/randomImage.js:1]: I see.

**Review by Tolfx — APPROVED** [review]
LGTM.

**Pull request merged**